PlastomeRegionBurstAndAlign.py

```diff
@@ -340,165 +322,154 @@ def main(args):
         if select_mode == 'int':
             ExtractAndCollect(main_d_nucl).do_INT(main_d_intron2, rec, log)
             main_d_nucl.update(main_d_intron2)
-        # Check if main dictionary empty
+
         if not main_d_nucl.items():
-            log.critical("No items in main dictionary" % out_dir)
+            log.critical("No items in main dictionary")
             raise Exception()
-    # --------------------------------------------------------------------------#
-    # REMOVE DUPLICATE ANNOTATIONS
-    # Note: Not sure why I have to run this removal twice, but not all
-    #       duplicates are removed first time around.
-    action = "removing duplicate annotations"
```

Comment:
Code to keep the user informed of what the software is doing via the logger should be maintained! Thus, please keep something like:

    log.info("removing duplicate annotations")